Makefile

```diff
@@ -0,0 +1,12 @@
+PODMAN=podman
+CONTAINER_IMG_NAME=frigate
+CONTAINER_IMG_TAG=latest
```

Comment:
shouldn't we pin this to a specific version instead of `latest`?

Reply:
we build the image, so it's always latest, we're not pulling anything

Reply:
that what happens when you review from top to bottom :-)